feathr_project/feathr/settings.py

```diff
@@ -20,6 +21,8 @@ def __init__(self,
         self.event_timestamp_column = event_timestamp_column
         self.timestamp_format = timestamp_format
         self.observation_path = observation_path
+        if observation_path.startswith("http"):
+            logger.warning("Your observation_path starts with http, which might not work in Spark. Consider using paths starting with wasb[s]/abfs[s]/s3.")
```

Comment:
spark is impl details that users should not know. Let's just say Http is not supported.